yaroslav.nazarenko/second/tamagochi.rb

```diff
@@ -0,0 +1,90 @@
+class Pet
+  attr_reader(:health, :fun, :energy)
```

Comment:
```suggestion
  attr_accessor :health, :fun, :energy
```

Reply:
you need not only reader but also writer, because you setting up instance variables here

Reply:
where did he use setters here, tho?

Reply:
@alexgonor

Reply:
`class Pet
  attr_reader :health
    def initialize
      self.health = 10
    end
end

Pet.new
NoMethodError (undefined method `health=' for #<Pet:0x0000560a51c04f98>)`

It's just because he use `@instance` variable its working now, in general you don't need even use attr_reader if you set and get all value via instance variables